spec/requests/groups_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe 'GroupsController', type: :request do
+  let :user do
+    User.find_or_create_by(email: 'mert@gmail.com') do |user|
+      user.name = 'mert'
+      user.password = Devise.friendly_token.first(8)
+      user.confirmed_at = Time.now
```

Comment:
- [ ] It would be better if we fix our failing test cases as seen in the screenshot below
![Screenshot from 2023-03-22 12-49-43](https://user-images.githubusercontent.com/46388113/227016429-786a2335-d6cc-477f-a70b-ea062a89e66d.png)

Reply:
Here is the link updating the tests:
[33b4a77](https://github.com/mkaya13/budget-app-rails/pull/24/commits/33b4a77076c97042d53d674224e839a1da53
<img width="415" alt="All-tests-passed" src="https://user-images.githubusercontent.com/53873573/227031909-b1a2d509-3dee-4ef4-b49f-272a502eb28a.png">
33c3)

SS for tests: